include/tss2/tss2_esys.h

```diff
@@ -9,7 +9,7 @@
 #include "tss2_tcti.h"
 #include "tss2_sys.h"
 
-#ifdef __cplusplus
+#ifdef _cplusplus
```

Comment:
This should be double "__"
https://gcc.gnu.org/onlinedocs/cpp/Standard-Predefined-Macros.html